### DISCUSSION TITLE
Registry and Keycloak Setup Issue in Apple Machine(Mac M1)

### DISCUSSION THREAD

**Discussion by shiva-rakshith** [Sunbird-RC/community#724] [Issues] [answered]
I am trying to setup registry in mac m1 machine using this [docker-componse file](https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/docker-compose.yml). Getting below error with registry(sunbird-rc-core) and keycloak(sunbird-rc-keycloak) images.

```
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested 0 ✔ Container sunbird-rc-keycloak-1
```
Do we need to use different docker images for apple machines?

**Comment by holashchand** [comment, marked as answer]
Hii @shiva-rakshith , The issue of keycloak image not working in Apple Silicon devices is now resolved.
Please use the below image for keycloak to make it work!! 
`ghcr.io/sunbird-rc/sunbird-rc-keycloak:v1.0.1-apple-silicon`

_Let me know if you are still facing this issue._

**Reply by tobianointing** [reply]
Hi @holashchand, I am still facing this issue

**Reply by pallakartheekreddy** [reply]
@tobianointing What issue are you facing? Could you please share the error details?
